pkg/generate/testdata/models/apis/sns/0000-00-00/generator.yaml

```diff
@@ -1,6 +1,7 @@
 resources:
   Topic:
     unpack_attributes_map:
+      set_attributes_single_attribute: true
```

Comment:
As the 'single attribute' behavior is related to operation on the resource, should the configuration be set at operation level in the config? The resultant generator.yaml would look like:
```
operations:
    SetTopicAttributes:
         single_attributes: true
```

Reply:
Yeah, it's an interesting question. I did it this way because this is more of a variation on the "UnpacksAttributesMap" behaviour than it is for anything else. That said, I'm going to be using the OperationConfig.CustomOperation functionality to sidestep the weirdness in the SetTopicAttributes API call entirely for now, so the point may be moot (for the timebeing at least)